tools/packaging/static-build/qemu/build-qemu.sh

```diff
@@ -16,7 +16,7 @@ kata_static_build_scripts="${kata_static_build_dir}/scripts"
 
 git clone --depth=1 "${QEMU_REPO}" qemu
```

Comment:
Maybe this is what is needed?
```suggestion
git clone --depth=1 --branch="${QEMU_BRANCH}" "${QEMU_REPO}" qemu
```